packages/graphiql-plugin-code-exporter/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@graphiql/plugin-code-exporter",
+  "version": "0.0.1",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/graphql/graphiql",
+    "directory": "packages/graphiql-plugin-code-exporter"
+  },
+  "author": "LekoArts",
+  "main": "dist/graphiql-plugin-code-exporter.cjs.js",
+  "module": "dist/graphiql-plugin-code-exporter.es.js",
+  "types": "types/index.d.ts",
+  "license": "MIT",
+  "keywords": [
+    "react",
+    "graphql",
+    "graphiql",
+    "plugin",
+    "explorer"
+  ],
+  "files": [
+    "dist",
+    "src",
+    "types"
+  ],
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc --emitDeclarationOnly && node resources/copy-types.mjs && vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@graphiql/react": "^0.13.0",
```

Comment:
As far as I can see, we're only using the `GraphiQLPlugin` type from `@graphiql/react`. If so, we should make this a dev-dependency.